Text improvements to Partial Hydration (#699)
withastro/docs: src/pages/en/core-concepts/partial-hydration.md

i18nReady: true
---

> **Astro generates every website with zero client-side JavaScript, by default.** Use any frontend UI component from our official integrations that you'd like ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) and [Lit](https://lit.dev/)) and Astro will automatically render it to HTML at build-time and strip away all JavaScript. This keeps every site fast by default.

Author: We could also just change "any" instead... 😅

> **Astro generates every website with zero client-side JavaScript, by default.** Use any frontend UI component from our official integrations that you'd like ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) and [Lit](https://lit.dev/)) and Astro will automatically render it to HTML at build-time and strip away all JavaScript. This keeps every site fast by default.

Reviewer: Good call! How about this then.

```suggestion
**Astro generates every website with zero client-side JavaScript, by default.** Use a frontend UI component built with ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) or [Lit](https://lit.dev/)) and Astro will automatically render it to HTML at build-time and strip away all JavaScript. This keeps every site fast by default.
```

> **Astro generates every website with zero client-side JavaScript, by default.** Use any frontend UI component from our official integrations that you'd like ([React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) and [Lit](https://lit.dev/)) and Astro will automatically render it to HTML at build-time and strip away all JavaScript. This keeps every site fast by default.

Reviewer: Oh, missed those in all the Markdown links. Yeah, in general I feel if we can avoid parentheses that’s good. (I had a teacher once who told us that if something needs to be in parentheses, it doesn’t need to be in the essay.)

```suggestion
**Astro generates every website with zero client-side JavaScript, by default.** Use a frontend UI component built with [React](https://reactjs.org/), [Preact](https://preactjs.com/), [Svelte](https://svelte.dev/), [Vue](https://vuejs.org/), [SolidJS](https://www.solidjs.com/), [AlpineJS](https://alpinejs.dev/) or [Lit](https://lit.dev/) and Astro will automatically render it to HTML at build-time and strip away all JavaScript. This keeps every site fast by default.
```
